### PR TITLE
core: frontend: Update gitignore with yarn files

### PR DESCRIPTION
**Pull request by patrickelectric**
It also helps the IDE to figure out where to search or not

Signed-off-by: Patrick José Pereira <patrickelectric@gmail.com>

### CODE REVIEW

File: core/frontend/.gitignore [hunk 1]
```diff
@@ -1,2 +1,5 @@
 node_modules/
-dist/
\ No newline at end of file
+dist/
+.yarn/
+yarn.lock
```
Comment (rafaellehmkuhl) [inline core/frontend/.gitignore:4]: The other two are fine, but yarn.lock [should be commited](https://classic.yarnpkg.com/blog/2016/11/24/lockfiles-for-all/). It ensures everyone uses the same set of dependencies.

**Review by rafaellehmkuhl — CHANGES_REQUESTED** [review]

File: core/frontend/.gitignore (hunk 1 above)
Comment (patrickelectric) [reply core/frontend/.gitignore:4]: It's commited and it's being tracked by git already, this just helps the IDE to avoid it as a search source.

Comment (rafaellehmkuhl) [reply core/frontend/.gitignore:4]: If someone clones the repository after this (addition to gitignore) and modify the yarn.lock file, will this change be tracked by git? Will this person be able to commit? I don't remember exactly how the git index/cache is transported to new clones.

Comment (patrickelectric) [reply core/frontend/.gitignore:4]: Yes, they are going to be tracked and you'll be able to see the changes and commit.

Comment (rafaellehmkuhl) [reply core/frontend/.gitignore:4]: Nice

**Pull request merged**